src/Catel.Core/Catel.Core.Shared/Logging/Listeners/SeqLogListener.cs

```diff
@@ -89,46 +85,43 @@ protected override string FormatLogEvent(ILog log, string message, LogEvent logE
                 var textWriter = new StringWriter();
                 textWriter.Write("{\"events\":[");
 
-                var logEntries = batchEntries.Select(
-                    batchEntry => FormatLogEvent(batchEntry.Log, batchEntry.Message, batchEntry.LogEvent, batchEntry.ExtraData, batchEntry.Data, FastDateTime.Now))
-                    .Aggregate((log1, log2) => string.Format("{0},{1}", log1, log2));
+                var logEntries = batchEntries.Select(batchEntry => FormatLogEvent(batchEntry.Log, batchEntry.Message, batchEntry.LogEvent, batchEntry.ExtraData, batchEntry.Data, FastDateTime.Now)).Aggregate((log1, log2) => string.Format("{0},{1}", log1, log2));
 
                 textWriter.Write(logEntries);
                 textWriter.Write("]}");
 
                 var message = textWriter.ToString();
 
-                lock (_lock)
+                var baseUri = ServerUrl;
+                if (!string.IsNullOrWhiteSpace(baseUri))
                 {
-                    if (_webClient == null)
+                    using (var webClient = new WebClient { Encoding = Encoding.UTF8 })
```

Comment:
I am not sure whether creating a new web client on every request is a good idea (performance wise). I think it's better to let the listener implement IDisposable?

Reply:
Fixed